packages/cli/src/formatters/junit.ts

```diff
@@ -50,8 +57,8 @@ export const junit: Formatter = (results, { failSeverity }) => {
         output += `<failure message="${xmlEscape(result.message)}">`;
```

Comment:
do we need `xmlEscape` here as well then?
if not, could just remove the call together with the function